packages/ihe-gateway-sdk/src/models/patient-discovery/patient.ts

```diff
@@ -27,7 +27,7 @@ export const personalIdentifierSchema = z.object({
 });
 export type PersonalIdentifier = z.infer<typeof personalIdentifierSchema>;
 
-export const genderSchema = z.enum(["male", "female", "other", "unknown", "undefined"]).optional();
+export const genderSchema = z.enum(["male", "female", "other", "unknown"]);
 export type Gender = z.infer<typeof genderSchema>;
```

Comment:
Side note: is this going away when we sunset the IHE GW v1? I see [here](https://github.com/metriport/metriport/pull/2263/files#diff-72cddf82d5a9414ff8dafb69ada711d7ff8f05c53185db8c0edf3a31dd997821R37) that we have a schema specific for CQ/HIE, so this is only used for the OSS API <> IHE GW protocol, is that right? Just came to attention b/c it just replicate FHIR's genders.

Reply:
Yes so we technically dont really need any of the zod schemas (just the types they created) anymore.

Reply:
Even the types, in theory we already have those for internal use (inside the API and API<>Lambdas) and for external use (outbound/inbound CQ), no?

Let's ticket it?